src/coreclr/src/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/FlowAnnotations.cs

```diff
@@ -533,6 +540,8 @@ private class TypeAnnotations
             private readonly FieldAnnotation[] _annotatedFields;
             private readonly DynamicallyAccessedMemberTypes[] _genericParameterAnnotations;
 
+            public bool IsDefault => _annotatedFields == null && _annotatedFields == null && _genericParameterAnnotations == null;
```

Comment:
Typo: This is testing `_annotatedFields` twice.

Reply:
Oops, good catch!